src/proj7BittingCerratoCohenEllmer/bantam/lexer/precusortokens/PrecursorMathToken.java

```diff
@@ -22,16 +22,19 @@ public PrecursorMathToken(Stack<Character> sc, int n, String s) {
     @Override
     public void pushChar(char c) {
         spellingStack.push(c);
-
-        if (Character.isWhitespace(c) ||
-                Character.isAlphabetic(c) ||
-                Character.isDigit(c) ||
-                c == ';') {
-            popLastBeforeCreation = true;
+        
```

Comment:
Is anything functionally different here? This is rather difficult to read and I'm struggling to figure out why this is better

Reply:
Yes this is functionally very different than the current implementation.

Dale noted that "&&<||," is scanned as 3 error tokens. I will push a mod that you will hopeful have an easier time getting